### DISCUSSION TITLE
Question about disallowing rest of ref to be forwarded to PanelGroup

### DISCUSSION THREAD

**Discussion by eiffel205566** [bvaughn/react-resizable-panels#351] [Q&A] [answered]
Hello Brian..

Would you mind shed some light how to set it up to allow PanelGroup to accept an external ref as PanelGroup is used as a container most of time and would really like to give it a ref and get a reference to it

**Comment by bvaughn** [comment, marked as answer]
A reference to what? The underlying DOM element?

Check out the discussion in #248 for the APIs this package exports for interacting with the DOM elements. (I don’t consider this a common use case for the library though.)

**Comment by eiffel205566** [comment]
thanks a lot, that did the trick
